src/stories/Introduction.mdx

```diff
@@ -0,0 +1,205 @@
+import { Meta } from "@storybook/blocks";
```

Comment:
todo: I believe you can delete this page, right? :)

Reply:
yep I did hehe, deleted it